samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages_eu.properties

```diff
@@ -570,3 +570,9 @@ savedMessage=Work is being saved.
 subMessage=Work is being submitted.
 hideMessage=Hide Time Remaining
 showMessage=Show Time Remaining
+
+# S2U-21 Safe Exam Browser
```

Comment:
@pablosanroman Could you help us here?

Reply:
subMessage= Lana bidaltzeko prozesuan dago
hideMessage= Ezkutatu gainerako denbora 
showMessage= Erakutsi gainerako denbora

@mpellicer
#  S2U-21 Azterketa seguruko nabigatzailea

Reply:
Hi Pablo, I was requesting the 'seb_' strings, those strings are not related.